android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -385,6 +388,7 @@ private void setupDialogButtons(final PrefsListItemWrapper item, final Dialog di
 		confirm.setOnClickListener(new OnClickListener() {
 			@Override
 			public void onClick(View v) {
+				// Sliders
```

Comment:
I recommend checking your editor's indentation settings. I presume errors like this stem from a mix of `TAB`s vs. `SPACE`s and/or different `TAB`-size settings. Ideally your editor would simply auto-detect the current source file's indentation mode and use that. Atom, for instance, covers that via this [add-on](https://atom.io/packages/auto-detect-indentation).